google-cloud-core/README.md

```diff
@@ -6,7 +6,14 @@ This library contains shared types, such as error classes, for the google-cloud
 
 ## Supported Ruby Versions
 
-This library is supported on Ruby 2.0+.
+This library is currently supported on Ruby 2.0+.
+
+However, Ruby 2.3 or later is strongly recommended, as earlier releases are
```

Comment:
"are or are nearing end-of-life" -> "are at or are nearing end-of-life" or "have reached or are nearing end-of-life"?

Reply:
changed to "have reached or are nearing end-of-life"